### DISCUSSION TITLE
Accepted Methodology/Data Providers For Marginal Carbon Intensity

### DISCUSSION THREAD

**Discussion by will-iamalpine** [Green-Software-Foundation/sci#57] [General]
I can think of five data providers for location-based marginal carbon intensity. This opens up some questions: 

- How do we account for discrepancies? 
- Is it in scope of the SCI to define acceptable sources fo for this information? 
- Do we require these data sources meet a given standard that someone else has defined? (e.g. World Resources Institute, RMI, GHG)

**Data providers:** 
1.	WattTime ( @Henry-WattTime) has validated through RMI (also WRI certified?), which can be found on our website on this page at the [Methodology and Validation section](https://www.watttime.org/aer/how-aer-works/)
2.	ElectricityMap  (WRI certified?) 
3.	REsurety  models transmission constraints to get a locational marginal emissions intensity figure per building. 
4.	WeDeex by CSN Energy tracks carbon intensity of grid
5.	e2Intel uses LEEM (Location Emission Estimation Methodology) - Liang Downey (MSFT) is trying to join GSF and bring LEEM into the discussion

**Comment by jawache** [comment]
This is a great question, in terms of specific language in the SCI we should only provide the requirements and constraints of the data source, and we should also ensure I think that our requirements and constraints allow a free source of data, to implement the SCI shouldn't require purchasing data otherwise it greatly limits adoption and might be seen as a commercial play.

- So one requirement is that it's marginal.
- Another requirement might be that it uses **at least** yearly regional averages.
- The certification is a good point, I'm in agreement with adding that as a requirement as long as there is a free data source of at least yearly regional averages that are certified, we can specify a list of approved certification authorities.

We should avoid listing a set of organisations that provide paid for services in this area, that feels like we are stepping into the anti-trust territory. We can have ancillary docs parralel to this one about how to implement the SCI which talks about that, but in the SCI itself, I think that's something we should avoid.
